src/SFA.DAS.FindEpao/InnerApi/Responses/GetStandardsExtendedListItem.cs

```diff
@@ -0,0 +1,15 @@
+using System;
+using System.Collections.Generic;
+
+namespace SFA.DAS.FindEpao.InnerApi.Responses
+{
+    public class GetStandardsExtendedListItem
+    {
+        public int LarsCode { get; set; }
+        public string status { get; set; }
```

Comment:
Pascal case, though i'm not sure Status is needed, once we've tweaked the dates on the other part.

Reply:
removed